src/components/pages/Home.jsx

```diff
@@ -15,6 +16,9 @@ export const Home = () => {
 				{/* not sure if i should use h1, h2, or h3 here */}
```

Comment:
do you need to create a heading? that's what the h tags are for! h1 is biggest, h2 is a little smaller than h1, and then h3 is a little smaller than h2.